website/src/_includes/section-menu/documentation.html

```diff
@@ -121,6 +121,14 @@
         <li><a href="{{ site.baseurl }}/documentation/programming-guide/#using-metrics">Using metrics in pipeline</a></li>
       </ul>
     </li>
+    <li class="section-nav-item--collapsible">
+      <span class="section-nav-list-title">Lifecycles</span>
+
+      <ul class="section-nav-list">
+        <li><a href="{{ site.baseurl }}/documentation/programming-guide/#dofn">DoFn lifecycle</a></li>
```

Comment:
I think this would fit in http://apache-beam-website-pull-requests.storage.googleapis.com/9807/documentation/programming-guide/index.html#pardo where it talks about the DoFn processing elements one at a time.

Reply:
Agree, but as the diagram talks also about instanciation, serializability, etc.. and in general about the DoFn in a whole, I'd like to put it in a separate section under pardo. WDYT?